### PR TITLE
Merge frontend branch into dev

### PR DESCRIPTION
**Pull request by annachiu7**
- Improve map display on search
- Select by map extent gives users more feedback
- Users can now clear selection by map extent using a new button
- Import/Export query updated
- About page updated
- Pretty print JSON feature collection output
- Fix inconsistency when turning results page
- Tooltips added for ambiguous buttons

### CODE REVIEW

File: webserver/pytreedb_server/templates/about.html [hunk 1]
```diff
@@ -46,26 +46,26 @@
...
+                    <p>The database may be used to populate synthetic forest stands of given species composition and stand density for laser scanning simulations or radiative transfer models. For example, tree point clouds can be loaded as voxel models in HELIOS++, 3DGeo’s general purpose LiDAR simulator, with the “xyzloader” and virtually scanned in a simulation. For more information on the LiDAR simulator, please visit <a class="inPLink" href="https://uni-heidelberg.de/helios">https://uni-heidelberg.de/helios</a>. </p>
+                    <p>In another application, the database may be used to select trees to insert to a 3D city model for analyses of solar potential or urban heat. </p>
+                    <p>pytreedb was developed in the SYSSIFOSS project by the 3DGeo Research Group of Heidelberg University and the Institute of Geography and Geoecology (IFGG) of Karlsruhe Institute of Technology (KIT). The project is funded by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) - – 411263134 / 2019-2022).</p>
```
Comment (han16nah) [inline webserver/pytreedb_server/templates/about.html:65]: Please remove one of the dashes

File: webserver/pytreedb_server/templates/about.html [hunk 2]
```diff
@@ -46,26 +46,26 @@
...
 
                     <h2 class="secondaryTitle">Adding Your Data</h2>
-                    <p>If you are in possession of laser scanning data of trees, and are able to fulfill the minimal requirements of the GeoJSON shown above, we welcome you to contribute your data to the database. We do not provide a hosting service, but allow integration of your metadata into the database, so that query results may come from multiple sources. This way, you have full control over the data at all times.</p>
+                    <p>If you are in possession of laser scanning data of trees and are able to fulfil the minimal requirements of the GeoJSON shown above, we welcome you to contribute your data to the database. We do not provide a hosting service, but allow integration of your metadata into the database, so that query results may come from multiple sources. This way, you have full control over the data at all times. You can also use this website as template to implement your own web frontend for your own dataset. Checkout out pytreedb on GitHub and learn how to deploy your webserver.</p>
```
Comment (han16nah) [inline webserver/pytreedb_server/templates/about.html:68]: Please change "Checkout out" to "Check out"

**Review by han16nah — CHANGES_REQUESTED** [review]
The language menu is still present on the About page and the Contact page, can you remove this?

File: webserver/pytreedb_server/templates/about.html [hunk 3]
```diff
@@ -62,10 +62,10 @@ <h1 class="pageTitle">About pytreedb</h2>
...
-                    <p>If you are in possession of laser scanning data of trees and are able to fulfil the minimal requirements of the GeoJSON shown above, we welcome you to contribute your data to the database. We do not provide a hosting service, but allow integration of your metadata into the database, so that query results may come from multiple sources. This way, you have full control over the data at all times. You can also use this website as template to implement your own web frontend for your own dataset. Checkout out pytreedb on GitHub and learn how to deploy your webserver.</p>
+                    <p>If you are in possession of laser scanning data of trees and are able to fulfil the minimal requirements of the GeoJSON shown above, we welcome you to contribute your data to the database. We do not provide a hosting service, but allow integration of your metadata into the database, so that query results may come from multiple sources. This way, you have full control over the data at all times. You can also use this website as template to implement your own web frontend for your own dataset. Checkout pytreedb on GitHub and learn how to deploy your webserver.</p>
```
Comment (han16nah) [inline webserver/pytreedb_server/templates/about.html:68]: I think "Check out" needs a space in between.

**Review by han16nah — CHANGES_REQUESTED** [review]
in `templates\index.html`, L 82, please change the download link to https://github.com/3dgeo-heidelberg/pytreedb/raw/main/data/test/geojsons.zip (we still have the old heibox link there, which is invalid by now).

Other wise, I can approve the PR :)

**Comment by annachiu7** [comment]
@han16nah Can you approve the PR now? :)

**Review by han16nah — APPROVED** [review]

**Comment by han16nah** [comment]
> @han16nah Can you approve the PR now? :)

yes!

**Pull request merged**